example-chat-task/conf.yml

```diff
@@ -52,7 +52,7 @@ ratings:
     choices: *choices
 
 # rename _mturk -> mturk to enable crowd backend
-_mturk: 
```

Comment:
I would revert this back to _mturk for the default setting because boteval doesn't assume that mturk will be used by default